temp#1.py

```diff
@@ -0,0 +1,131 @@
+# Calista Adler and Brandon Zhao
+
+import pandas as pd
+import csv 
+
+# opening and reading data from records.csv file 
+data = pd.read_csv(r'/Users/calistaadler/Downloads/records.csv')
+
+# writing into new csv file 
+f = open('/Users/calistaadler/Downloads/tempout.csv', 'w')
+writer = csv.writer(f)
+
+# creates column titles 
+header = ['Timestamp', 'ModA Temp', 'ModB Temp', 'ModC Temp', 'Gate Driver Board Temp']
+writer.writerow(header)
+
+# used to parse through each row 
+i = 0
+n = 1
+
+# id for temperature #1
+temp1_id = '00A0'
+
+# replaces square brackets and single quotes for correct formatting and type conversions
+def replace(var):
+    var = var.replace("]","")
+    var = var.replace("[","")
+    var = var.replace("'","")
+
+# iterates through each row in original records file
+for index, row in data.iterrows():
+    
+    year = data['year'][i:n].tolist()
```

Comment:
Now that you are using `data.iterrows()`, `row` will point to a `pd.Series` (essentially, a python dictionary) containing the value in that row for each column. Thus, you should be able to replace
`year = data['year'][i:n].tolist()`
with
`year = row['year']`
This should also make the `replace()` function unnecessary because the brackets and commas were only there because you converted it to a list.